scripts/airflow3/setup.sh

```diff
@@ -4,26 +4,17 @@
 set -e
 
 # Create a UV virtual environment named 'env' (you can change this as needed)
-echo "Creating UV virtual environment at $(pwd)/tools"
-uv venv venv --directory "$(pwd)/scripts/airflow3"
+echo "Creating virtual environment at $(pwd)/tools"
+python3 -m venv "$(pwd)/scripts/airflow3/venv"
 
 # Activate the virtual environment
 echo "Activating virtual environment..."
 source "$(pwd)/scripts/airflow3/venv/bin/activate"
 
 # Install dependencies in the virtual environment
 echo "Installing dependencies..."
-uv pip install -r "$(pwd)/scripts/airflow3/requirements.txt"
+pip3 install --pre -r "$(pwd)/scripts/airflow3/requirements.txt"
```

Comment:
Once we need to re-add the providers, we can leverage the constraints file eg.
```
--constraints https://raw.githubusercontent.com/apache/airflow/refs/tags/constraints-3.0.0rc1.post4/constraints-3.9.txt
```

Since this is not needed now, we probably can do it only once we need